src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![deny(missing_docs)]
```

Comment:
I wouldn't use `deny` as deny prevents building iirc.

I would just use `warn` and then the linter will elevate these to an error in CI

Reply:
I usually recommend this:
```rust
#![warn(missing_docs, clippy::missing_docs_in_private_items)]
```